### ISSUE TITLE
autokey not installing with gentoo when following wiki

### ISSUE THREAD

**Issue by tjallingk** [autokey/autokey#926]
### AutoKey is a Xorg application and will not function in a Wayland session. Do you use Xorg (X11) or Wayland?

Xorg

### Has this issue already been reported?

- [X] I have searched through the existing issues.

### Is this a question rather than an issue?

- [X] This is not a question.

### What type of issue is this?

Documentation

### Choose one or more terms that describe this issue:

- [ ] autokey triggers
- [ ] autokey-gtk
- [ ] autokey-qt
- [ ] beta
- [ ] bug
- [ ] critical
- [ ] development
- [X] documentation
- [ ] enhancement
- [X] installation/configuration
- [ ] phrase expansion
- [ ] scripting
- [ ] technical debt
- [ ] user interface

### Other terms that describe this issue if not provided above:

emerge,gentoo

### Which Linux distribution did you use?

gentoo with kde plasma and x11

### Which AutoKey GUI did you use?

None

### Which AutoKey version did you use?

newest trying to install but non used

### How did you install AutoKey?

according to the limited documentation i found for gentoo on the wiki with emerge(gentoo package manager) with the following command `emerge --noreplace --ask dbus-python pyinotify python-xlib qscintilla-python wmctrl` 

### Can you briefly describe the issue?

I think there is something missing from the documentation as i read something about app-misc/autokey-gtk but that one is not found nor is it in the limited documentation. Maybe it has moved then maybe add the new location to the command but it now only seems to install needed dependencies not the program itself.   

### Can the issue be reproduced?

Always

### What are the steps to reproduce the issue?

1 read documentation and wiki for the install with gentoo
2 execute the command given by the wiki for gentoo `emerge --noreplace --ask dbus-python pyinotify python-xlib qscintilla-python wmctrl`
3 try and open autokey-gtk and see it missing

### What should have happened?

i did expect to be atleast able to open autokey from cli that there is no icon etc is a con but not a breaking feature but i expect to install the application when i exactly follow the wiki

### What actually happened?

a whole lot of nothing only some dependency installation

### Can you provide the output of the AutoKey command?

```bash
autokey-gtk --verbose       bash: autokey-gtk: command not found
autokey-qt --verbos         bash: autokey-qt: command not found
```

### Anything else?

i am currently trying to install it with pip in a python venv ( but that is only a temporary workaround if it works even as i am missing a ton of dependencies  as my python returns error: externally-managed-environment when trying it without the virtual environment

**Comment by josephj11** [comment]
I haven't heard from any Gentoo users in quite a while. I'll ask for help on our two main support forums and see if anyone responds.

We also have a major shortage of developers ATM, < 1, so getting answers to advanced issues and getting anything fixed is a problem.

**Comment by josephj11** [comment]
Is this resolved? Can it be closed?

**Comment by gr3m1in** [comment]
This repo does not provide any ebuild (gentoo package building file) for autokey, and the documentation just shows how to install dependencies, not autokey itself.
There is a thread on gentoo forums regarding it
https://forums.gentoo.org/viewtopic-p-7046610.html#7046610
and at the end of the thread, user **teika** has proposed **autokey-0.90.4.ebuild**
(however that ebuild is VERY old and will likely not work out-of-the-box as it relies on obsolete EAPI=4 version)

**Comment by gr3m1in** [comment]
> Is this resolved? Can it be closed?

IMO it is worth to mention in the docs, that the repo does not provide any gentoo ebuild and the provided command is only for deps on gentoo, to avoid further duplicates of this issue...

Or to add an ebuild to the repo, which looks expected as gentoo distro is mentioned in the docs

**Comment by josephj11** [comment]
I don't think we have any Gentoo expertise in this project. If you get an ebuild or whatever to work, feel free to contribute to our wiki. Anyone with a GitHub account has full wiki access.

**Comment by tjallingk** [comment]
Thanks for clearing this up this can be closed and I think it might be infact a good idea to or remove the part from wiki or to clarify that it only installs dependencies

**Comment by josephj11** [comment]
I added a line to the wiki asking for Gentoo users to improve that section. It also doesn't address the optional dependencies like visgrep (but no one uses visgrep AFAIK). I would leave this open because it's a documentation bug at minimum.

**Comment by develBuzz** [comment]
Hey,
i've added some information about the GTK installation for gentoo. This should work out of the box.

**Comment by josephj11** [comment]
Thank you. Looks very detailed (as it should be). I think this issue can be closeed now.

**Comment by mjeveritt** [comment]
Is this still an issue? I do have some Gentoo expertise, and can probably help poke things in the right direction 😀

**Comment by josephj11** [comment]
@mjeveritt Thanks. Looks like it was a documentation problem. If you want to help with it, you can review the [wiki](https://github.com/autokey/autokey/wiki/Installing#gentoo-linux) and update it as necessary. IDK anything about ebuild... or what, if anything should be added to support it.
